p2719.md

```diff
@@ -432,134 +474,157 @@ Updates to [expr.new]{.sref}
 >    consideration.
 
 ::: add
->  - [11.2]{.pnum} If any of the deallocation candidates have a first parameter of the same type
+>  - [11.2]{.pnum} If any of the deallocation functions have a first parameter of the same type
 >    as the type-identity, all deallocation functions that do not have a first parameter
 >    of that type are eliminated from further consideration.
 :::
 
->  - [9.2->11.3]{.pnum} If the type has new-extended alignment, a function with a parameter of type
+>  - [11.3]{.pnum old=9.2} If the type has new-extended alignment, a function with a parameter of type
 >    std​::​align_val_t is preferred; otherwise a function without such a parameter is preferred. If
 >    any preferred functions are found, all non-preferred functions are eliminated from further
 >    consideration.
+
+::: add
+>  - [11.4]{.pnum} If multiple functions of the same type are found, only the most viable functions
+>    [over.match.best]{.unnumbered .sref} are retained, all others are eliminated from further
+>    consideration.
+:::
+
 >
 > ...
 >
-> [10->12]{.pnum} [If the first parameter of the selected deallocation function is the same type as
-> the type-identity, the type-identity is passed as the first argument to the deallocation call.]{.add}
-> For a single-object delete expression, the deleted object is the object A pointed to by the operand
-> if the static type of _A_ does not have a virtual destructor, and the most-derived object of _A_
-> otherwise.
-> ...
+> [12]{.pnum old=10} For a single-object delete expression, the deleted object is the object A pointed
+> to by the operand if the static type of A does not have a virtual destructor, and the most-derived
+> object of A otherwise. [If the deallocation function is not a destroying operator delete and the
+> deleted object is not the most derived object in the former case, the behavior is undefined, as
+> stated above.]{.note}
+> [If the selected function is type aware, then it will be called with the type-identity argument
+> as the first argument to the call]{.add}. For an array delete expression, the deleted object is
+> the array object. When a delete-expression is executed, the selected deallocation function shall
+> be called with the address of the deleted object in a single-object delete expression, or the
+> address of the deleted object suitably adjusted for the array allocation overhead ([expr.new]) in
+> an array delete expression, as [the]{.add}[its]{.rm} first argument [following the type identity
+> argument if present]{.add}.
+
+## expr.const
+
+[Supporting type aware new and delete in constant expressions was discussed, but this section was
+not present in the drafts presented at the Wroclaw meeting. For system allocators this is something
+that needs to be supported. In principle vendor extensions would be able support the required use
+case pending a more universally applicable constexpr operator new/operator delete story.]{.draftnote audience=EWG}
```

Comment:
This drafting note is unnecessary. The right way to call out that we want this recorded explicitly is to add it to the design choices like you've done below, and to get re-scheduled in EWG just for confirm that choice.

Reply:
- [ ] todo